dropshot/tests/test_config.rs

```diff
@@ -125,8 +125,6 @@ where
     fn make_client(&self) -> hyper::Client<C>;
     fn make_server(&self, bind_port: u16) -> HttpServer<Self::Context>;
     fn make_uri(&self, bind_port: u16) -> hyper::Uri;
-
-    fn log(&self) -> &slog::Logger;
```

Comment:
@davepacheco any reason to keep this? I can't see how it might be in use.

Reply:
I can't see one.